state/block_test.go

```diff
@@ -0,0 +1,99 @@
+package state
+
+import (
+	"fmt"
+	"math/big"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/core/types"
+)
+
+func TestAddBlock(t *testing.T) {
+	dataDir := "../test_data/block"
```

Comment:
Could we use ioutils tempdir instead and replace this part till line 31 including the defer statement, for this instead: 

```
	dataDir, err := ioutil.TempDir("", "block_test")
	if err != nil {
		t.Error("Failed to create temp folder for block_test", "err", err)
		return
	}

	//Create a new blockState
	blockState, err := NewBlockState(dataDir)
	if err != nil {
		t.Fatal(err)
	}
```

that would be a bit simpler, what do you think ?

Reply:
done